modules/Core/pages/complete_signup.php

```diff
@@ -74,7 +74,7 @@
                     EventHandler::executeEvent('validateUser', [
                         'user_id' => $target_user->data()->id,
                         'username' => $target_user->getDisplayname(),
-                        'content' => str_replace('{x}', $target_user->getDisplayname(), $language->get('user', 'user_x_has_validated')),
+                        'content' => $language->get('user', 'user_x_has_validated', ['user' => $target_user->getDisplayname()]),
```

Comment:
Is it `getDisplayname()` or `getDisplayName()`?

Reply:
The function declaration is “getDisplayName” :p

Reply:
declaration used getDisplayName but entire NamelessMC and Modules uses getDisplayname